package.json

```diff
@@ -5,54 +5,51 @@
     "start": "ng serve demo",
     "build:lib": "npm run copy:readme && ng build mat-file-upload",
     "build:demo": "ng build demo --prod",
-    "copy:readme": "copyfiles README.md projects/mat-file-upload",
+    "copy:readme": "cp README.md projects/mat-file-upload",

```

Comment:
`copyfiles` is used in place of `cp` for cross-platform compatibility

Reply:
Just trying to think about not introducing the dependency… you could do `cp README.md projects/mat-file-upload || copy README.md projects\\mat-file-upload`, or even switch to Node.js `node -e \"require(require('fs').copyFileSync('README.md', require('path').join('projects', 'mat-file-upload'));`

Reply:
Just trying to think about not introducing the dependency… you could do `cp README.md projects/mat-file-upload || copy README.md projects\\mat-file-upload`, or even switch to Node.js `node -e \"require(require('fs').copyFileSync('README.md', require('path').join('projects', 'mat-file-upload'));\"`

Reply:
Since the dependency is only required to build & package the library (not for any consumers of the npm package), I would prefer to leave it as-is.